java-containeranalysis/.OwlBot-hermetic.yaml

```diff
@@ -34,5 +33,11 @@ deep-copy-regex:
   dest: "/owl-bot-staging/java-containeranalysis/v1/grpc-google-cloud-containeranalysis-v1/src"
 - source: "/google/devtools/containeranalysis/v1/.*-java/gapic-google-.*/src"
   dest: "/owl-bot-staging/java-containeranalysis/v1/google-cloud-containeranalysis/src"
+- source: "/google/devtools/containeranalysis/v1beta1/.*-java/proto-google-.*/src"
+  dest: "/owl-bot-staging/java-containeranalysis/v1beta1/proto-google-cloud-containeranalysis-v1beta1/src"
+- source: "/google/devtools/containeranalysis/v1beta1/.*-java/grpc-google-.*/src"
+  dest: "/owl-bot-staging/java-containeranalysis/v1beta1/grpc-google-cloud-containeranalysis-v1beta1/src"
+- source: "/google/devtools/containeranalysis/v1beta1/.*-java/gapic-google-.*/src"
+  dest: "/owl-bot-staging/java-containeranalysis/v1beta1/google-cloud-containeranalysis/src"
```

Comment:
Could you refactor this part to remove the version, just like [this](https://github.com/googleapis/google-cloud-java/blob/8cac6d6ecfbbed3ca74698b49432363ac43d9c5a/java-errorreporting/.OwlBot-hermetic.yaml#L27-L35)

Reply:
Updated